extensions/azdata/src/common/httpClient.ts

```diff
@@ -35,6 +37,20 @@ export namespace HttpClient {
 						outputChannel.appendLine(response.statusMessage);
 						return reject(response.statusMessage);
 					}
+					const filename = path.basename(response.request.path);
+					const targetPath = path.join(targetFolder, filename);
+					outputChannel.appendLine(loc.downloadingTo(filename, targetPath));
+					// Wait to create the WriteStream until here so we can use the actual
+					// filename based off of the URI.
+					downloadRequest.pipe(fs.createWriteStream(targetPath))
+						.on('close', async () => {
+							outputChannel.appendLine(loc.downloadFinished);
+							resolve(targetPath);
+						})
+						.on('error', (downloadError) => {
+							reject(downloadError);
+							downloadRequest.abort();
```

Comment:
Old code logs the downloadError to output channel, but the new code does not? Can you comment on this removal?